source/_integrations/zwave_js.markdown

```diff
@@ -44,18 +44,56 @@ ha_integration_type: hub
 ha_zeroconf: true
 ---
 
-This integration allows you to control a Z-Wave network via the [Z-Wave JS](https://zwave-js.github.io/node-zwave-js/#/) driver.
+The Z-Wave integration allows you to control a Z-Wave network via the [Z-Wave JS](https://zwave-js.github.io/node-zwave-js/#/) driver.
 
-To Run Z-Wave you will need a [Supported Z-Wave dongle](/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules), a running Z-Wave JS server (using only **one** of the add-ons or installation methods described below), and the Z-Wave integration.
+## Overview
 
-{% include integrations/config_flow.md %}
+To run a Z-Wave network, you need the following elements:
 
-If you run Home Assistant Container, Home Assistant Core, or you don't want to use the built-in Z-Wave JS Server add-on, please see the [advanced installation instructions](#advanced-installation-instructions).
+* A [supported Z-Wave controller](/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules). First time user? For recommendations on what to buy, go [here](integrations/zwave_js/#which-z-wave-controller-should-i-buy).
+* A running Z-Wave JS server.
+* An installed Z-Wave integration in Home Assistant.
 
-For new installations, network security keys will be automatically generated for you. If this Z-Wave stick has already been paired with secure devices, you need to enter the previously used network key as the S0 network key. S2 security keys will be automatically generated for you. Make sure that you keep a backup of these keys in a safe place in case you need to move your Z-Wave stick to another device.
+## Setting up a Z-Wave JS server
+
+The easiest way to get started is by using the built-in Z-Wave JS add-on in Home Assistant.
+
+For other ways to setup a Z-Wave server, refer to the [advanced installation instructions](#advanced-installation-instructions).
+
+Follow these steps:
+
+1. Open the Home Assistant user interface.
+1. Plug the Z-Wave dongle into the device running Home Assistant.
+   * Most likely, your dongle will be recognized automatically. On the user interface, you will be asked if you want to set up this device with the Z-Wave JS add-on. Select **Submit**.
+   * If your dongle is not recognized, follow these steps:
+
+{% details "Manual setup steps" %}
+Use this My button:
+
+{% my config_flow_start badge domain="zwave_js" %}, or follow these steps:
+
+* Browse to your Home Assistant instance.
+* In the sidebar click on _**{% my config icon %}**_.
+* From the configuration menu select: _**{% my integrations %}**_.
+* In the bottom right, click on the
+  _**{% my config_flow_start icon domain="zwave_js" %}**_ button.
+* From the list, search and select _**"Z-Wave"**_.
+* Follow the instructions on screen to complete the set up.
+
+{% enddetails %}
+
+3. Wait for the installation to complete.
```

Comment:
Something happend with the numbering here

Reply:
I was trying to use a workaround. If I put 1. here, the numbering starts over:

![image](https://user-images.githubusercontent.com/38767475/230018212-52648f53-8442-466a-a755-d4eac76e64ef.png)